### PR TITLE
Uses SES V2 APIs to manage EmailTemplates & CustomVerificationEmailTemplates

### PR DESCRIPTION
**Pull request by paynegreen**
This implements the v2 APIs for the following functions:
### EmailTemplate
* `get_email_template/1`
* `list_email_templates/1`
* `create_email_template/2`
* `update_email_template/4`
* `delete_email_template/1`
* `test_render_email_template/2`
### CustomVerificationEmailTemplate
* `list_custom_verification_email_templates_v2/1`
* `get_custom_verification_email_templates_v2/1`
* `create_custom_verification_email_template_v2/6`
* `update_custom_verification_email_template_v2/2`
* `delete_custom_verification_email_template_v2/1`
* `send_custom_verification_email/3`

### CODE REVIEW

**Comment by lessless** [comment]
Thanks,

I'm going to take a look later this week.

File: lib/ex_aws/ses.ex [hunk 1]
```diff
@@ -355,6 +375,28 @@ defmodule ExAws.SES do
...
+        "Subject" => subject
+      }
+      |> put_if_not_nil("Html", html)
+      |> put_if_not_nil("Text", text)
+
+    params =
+      %{
+        "TemplateName" => template_name
+      }
+      |> Map.put("TemplateContent", template_content)
```
Comment (lessless) [inline lib/ex_aws/ses.ex]: I don't see a sense to use two different methods to add elements to a map. 

Can the same thing be accomplished with: 
```suggestion
    params =
      %{
        "TemplateName" => template_name,
        "TemplateContent" => template_content
      }
```

**Review by lessless — COMMENTED** [review]
Looks good, just one minor comment

File: lib/ex_aws/ses.ex (hunk 1 above)
Comment (paynegreen) [reply lib/ex_aws/ses.ex]: I agree.

**Review by lessless — APPROVED** [review]
Thanks 🎉

**Pull request merged**